core/deployment/src/main/java/io/quarkus/deployment/CodeGenerator.java

```diff
@@ -26,6 +26,7 @@ public static void initAndRun(ClassLoader classLoader,
             AppModel appModel) throws CodeGenException {
         List<CodeGenData> generators = init(classLoader, sourceParentDirs, generatedSourcesDir, buildDir, sourceRegistrar);
         for (CodeGenData generator : generators) {
+            generator.setRedirectIO(true);
```

Comment:
Does this `initAndRun` get called _only_ for Gradle build? In which case this change looks good. The reason why the `ProcessUtil` has been conditionally used is to avoid streaming the STDOUT and STDERR when Maven tool is being used and instead just rely on inheriting IO. So as long as this gets called only for Gradle, the change looks fine to me.

Reply:
Yes, I ran a search on all the project and this is only used by gradle. Maven OTOH, uses the `init` and the `trigger` method.

Reply:
Thank you for confirming that.